source/fundamentals/atlas-search.txt

```diff
@@ -0,0 +1,608 @@
+.. _csharp-atlas-search:
+
+============
```

Comment:
It would be great to have few more advanced examples:
1. [Multipath](https://www.mongodb.com/docs/atlas/atlas-search/path-construction/#std-label-ref-path)
2. [Score function](https://www.mongodb.com/docs/atlas/atlas-search/scoring/#std-label-scoring-ref)
3. [Facets](https://www.mongodb.com/docs/atlas/atlas-search/facet/) 

I think this can be done later in follow up ticket.